pom.xml

```diff
@@ -263,6 +263,57 @@
             <scope>runtime</scope>
         </dependency>
 
+        <!--SOCIAL DEPENDENCIES START-->
```

Comment:
Holy shit, lot of dependencies!!
I got afraid. Will see later when I get some time for coffee.
Thanks for the contribution 😄